flytekit/tools/fast_registration.py

```diff
@@ -50,7 +51,7 @@ def get_additional_distribution_loc(remote_location: str, identifier: str) -> st
     :param Text identifier:
     :return Text:
     """
-    return _os.path.join(remote_location, "{}.{}".format(identifier, "tar.gz"))
+    return posixpath.join(remote_location, "{}.{}".format(identifier, "tar.gz"))
```

Comment:
why do we need this?

Reply:
when we use `os.path.join` on Windows, it will use `\\` to join these two paths. However, It doesn't work for the remote path. For example, the s3 path will become something like `s3://bucket/test\\key`